chain/types/bigint.go

```diff
@@ -63,6 +63,10 @@ func BigCmp(a, b BigInt) int {
 	return a.Int.Cmp(b.Int)
 }
 
+func BigPos(bi BigInt) bool {
```

Comment:
I think [`Sign()`](https://golang.org/pkg/math/big/) might make this redundant.

Reply:
Sorry I took a look at the link but it wasn't directly obvious how that would help. Will it tell me if a given int is positive?

Reply:
[Does this help?](https://golang.org/pkg/math/big/#Int.Sign) I think it'll work but I haven't tried it. Barring that, after looking at the source code a little more closely, I think you can just use `bi.neg` would give you a boolean indicating if it's negative. If neither one of them works, then yeah, your code's probably as good as it gets.

Reply:
Yeah, just use `.Sign()` for this. if it returns a number greater than zero, its positive. If it returns zero, its zero, if it returns a negative number, its negative